### DISCUSSION TITLE
[🇪🇸] Cambiar el tooltip de "turno base" en la cabecera de la ficha por "turno natural"

### DISCUSSION THREAD

**Discussion by Linkaynn** [AnimaBeyondDevelop/AnimaBeyondFoundry#55] [Ideas]
*Esta issue ha sido trasladada desde el antiguo sistema de tareas*

Cambiar el tooltip de "turno base" en la cabecera de la ficha por "turno natural"

**Comment by DavidDPG** [comment]
Si nos guiamos por cómo se montan otras fichas de Anima, este cambio implica que el valor al que este tooltip se refiera sería el turno base + el bonificador de desarmado. Esto está recogido en la página 16 del Core Exxet.

O bien el valor ya ha sido cambiado o habría que cambiarlo.

**Comment by Guote** [comment]
Sí, actualmente en ese campo se escribe el turno desarmado/natural. Se trataría de corregir el tooltip, pero aunque "turno natural" sea lo que se usa en otras fichas, quizás "turno desarmado" sea más user-friendly. Las dos serían correctas de todos modos
